apps/frontend/src/components/fap/Proposals/FapProposalsAndAssignmentsTable.tsx

```diff
@@ -355,10 +355,11 @@ const FapProposalsAndAssignmentsTable = ({
         continue;
       }
 
-      allProposalReviews.push(...proposalReviews);
-      allProposalReviews.map(
-        (proposalReview) => (proposalReview.proposalPk = proposalPk)
+      const proposalReviewsWithProposalPk = proposalReviews.map(
+        (proposalReview) => ({ ...proposalReview, proposalPk })
       );
+
+      allProposalReviews.push(...proposalReviewsWithProposalPk);
```

Comment:
I think we could simplify away the intermediate array
```suggestion
      proposalReviews.forEach(
        (proposalReview) => allProposalReviews.push({ ...proposalReview, proposalPk })
      );
```

Reply:
testing locally looks good